src/engine/source/metrics/src/providerHandler.cpp

```diff
@@ -33,39 +33,8 @@ void ProviderHandler::create(std::shared_ptr<MetricsContext> data)
             data->meterProvider->AddMetricReader(std::move(data->reader));
-            switch (data->instrumentType)
-            {
-                case opentelemetry::sdk::metrics::InstrumentType::kHistogram:
-                {
-                    std::string version{"1.2.0"};
-                    std::string schema{"https://opentelemetry.io/schemas/1.2.0"};
-                    auto name = data->histogramName + "_histogram";
-                    auto instrumentSelector = std::unique_ptr<opentelemetry::sdk::metrics::InstrumentSelector>(new opentelemetry::sdk::metrics::InstrumentSelector(data->instrumentType, name));
-                    auto meterSelector = std::unique_ptr<opentelemetry::sdk::metrics::MeterSelector>(new opentelemetry::sdk::metrics::MeterSelector(data->histogramName, version, schema));
-                    std::shared_ptr<opentelemetry::sdk::metrics::AggregationConfig> aggregation_config {new opentelemetry::sdk::metrics::HistogramAggregationConfig};
-                    static_cast<opentelemetry::sdk::metrics::HistogramAggregationConfig *>(aggregation_config.get())->boundaries_ = data->histogramVector;
-                    std::unique_ptr<opentelemetry::sdk::metrics::View> histogramView{new opentelemetry::sdk::metrics::View{
-                    data->histogramName, "description", opentelemetry::sdk::metrics::AggregationType::kHistogram, aggregation_config}};
-                    data->meterProvider->AddView(std::move(instrumentSelector), std::move(meterSelector), std::move(histogramView));
-                    opentelemetry::metrics::Provider::SetMeterProvider(provider);
-                    break;
-                }
-                case opentelemetry::sdk::metrics::InstrumentType::kCounter:
-                {
-                    std::string version{"1.2.0"};
-                    std::string schema{"https://opentelemetry.io/schemas/1.2.0"};
-                    auto name = data->counterName + "_counter";
-                    auto instrumentSelector = std::unique_ptr<opentelemetry::sdk::metrics::InstrumentSelector>(new opentelemetry::sdk::metrics::InstrumentSelector(data->instrumentType, name));
-                    auto meterSelector = std::unique_ptr<opentelemetry::sdk::metrics::MeterSelector>(new opentelemetry::sdk::metrics::MeterSelector(data->counterName, version, schema));
-                    std::unique_ptr<opentelemetry::sdk::metrics::View> counterView{new opentelemetry::sdk::metrics::View{
-                    data->counterName, "description", opentelemetry::sdk::metrics::AggregationType::kDefault}};
-                    data->meterProvider->AddView(std::move(instrumentSelector), std::move(meterSelector), std::move(counterView));
-                    opentelemetry::metrics::Provider::SetMeterProvider(provider);
-                    break;
-                }
-                default:
-                    break;
-            }
+            opentelemetry::metrics::Provider::SetMeterProvider(provider);
```

Comment:
Add the boundaries to build the frecuency. See: https://github.com/open-telemetry/opentelemetry-cpp/blob/7c0826efe0cf205f1f30d6c6a9d0d3a0254adb92/examples/metrics_simple/metrics_ostream.cc#L76

Reply:
It only works if an aggregation is made, in other words, if a view is added, which is optional. It's worth it?